### PR TITLE
Reduce duplicated code for banners

### PR DESCRIPTION
**Pull request by rauhul**
Unifies the banner and cta css styles used on multiple pages.

Moves the element to a common location so the it can be found in the
source easily and be reused across pages.

### CODE REVIEW

**Comment by rauhul** [comment]
@swift-ci test

File: assets/stylesheets/_elements.scss [hunk 1]
```diff
@@ -0,0 +1 @@
+@import "elements/banner";
```
Comment (rauhul) [inline assets/stylesheets/_elements.scss:1]: Note I plan on moving the grid elements from #742 into this folder

**Comment by rauhul** [comment]
@swift-ci test

**Comment by dempseyatgithub** [comment]
Do the cherry-picks in this PR include turning the banners on all the pages orange?

**Comment by rauhul** [comment]
@dempseyatgithub nope, I split out the orange for ".banner.primary" which is only used on the home page right now.

**Review by kaishin — APPROVED** [review]

**Comment by kaishin** [comment]
@rauhul Feel free to merge.

**Comment by rauhul** [comment]
@kaishin unfortunately I cannot merge because I lost "write" and "push" access to this repo after the move to "swiftlang". 

@swiftlang/contributor-experience how can I regain access?

**Comment by xedin** [comment]
@rauhul I think you want to reach out to the website workgroup on forums.

**Comment by rauhul** [comment]
> @rauhul I think you want to reach out to the website workgroup on forums.

Thanks I started a forum group dm!

**Comment by kaishin** [comment]
Sorry @rauhul I wasn't aware. I will take care of merging in the meantime then 👍

**Pull request merged**